tests/Server.php

```diff
@@ -75,6 +75,32 @@ public static function enqueue($responses)
         ]);
     }
 
+    /**
+     * Queue a single raw response manually, to handle cases where PSR7 response is not suitable.
+     *
+     * @param int|string  $statusCode   Status code for the response, e.g. 200
+     * @param string      $reasonPhrase Status reason response e.g "OK"
+     * @param array       $headers      Array of headers to send in response
+     * @param string|null $body         Body to send in response
+     *
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public static function enqueueRaw($statusCode, $reasonPhrase, $headers, $body)
+    {
+        $data = [
+            [
+                'status'  => (string) $statusCode,
+                'reason'  => $reasonPhrase,
+                'headers' => $headers,
+                'body'    => \base64_encode((string) $body)
+            ]
+        ];
+
+        self::getClient()->request('PUT', 'guzzle-server/responses', [
+            'json' => $data
```

Comment:
Missing trailing comma.

Reply:
This doesn't seem to follow the standard set with the method above this (enqueue) which doesn't use trailing commas on arrays for a very similar request - just wanted to confirm that?

Reply:
As long as the CI passes, then Im fine. 

We should consider update our CS rules if the CI is green with the current code (But that is a separate PR)